doc/API.md

```diff
@@ -1087,7 +1085,6 @@ Response:
 	UploadProgress float32
 	Nickname       string
 	Filesize       uint64
-	TimeRemaining  types.BlockHeight (uint64)
```

Comment:
Kay, reflected.

Shouldn't there be a `UploadHeight` or something to that effect to help determine `TimeRemaining` and/or things like: total rented time, total rent paid, etc?